testing/tests/values.rs

```diff
@@ -88,3 +88,23 @@ fn test_value_function_getter() {
     values.insert("a".to_string(), Box::new(false));
     assert_eq!(V.render_with_values(&values).unwrap(), "");
 }
+
+#[test]
+fn test_value_in_subtemplates() {
```

Comment:
We should take the habit to explain what the test is checking. Do you mind adding a code comment please?

Reply:
Yup, good call! Added.